new-lamassu-admin/src/pages/Wallet/AdvancedWallet.js

```diff
@@ -0,0 +1,63 @@
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+import * as R from 'ramda'
+import React from 'react'
+
+import { NamespacedTable as EditableTable } from 'src/components/editableTable'
+import { fromNamespace, toNamespace, namespaces } from 'src/utils/config'
+
+import { AdvancedWalletSchema, getAdvancedWalletElements } from './helper'
+
+const { utils: coinUtils } = require('lamassu-coins')
```

Comment:
Shouldn't this import use ES6 import keyword? Surprised the linter hasn't thrown an error

```suggestion
import { utils as coinUtils } from 'lamassu-coins'
```

Reply:
good point, don't know what happened